src/PhotoNim.nim

```diff
@@ -80,8 +105,92 @@ if args["pfm2png"]:
     
     pfm2png(fileIn, fileOut, alpha, gamma)
 
+#-----------------------------#
+#       Demo executable       #
+#-----------------------------#
+
 elif args["demo"]: 
-    quit "Eh volevi"
+    var 
+        height = 1000
+        width = 1600
+
+    if args["--width"]: 
+        try: width = parseInt($args["--width"]) 
+        except: echo "Warning: width must be an integer. Default value is used."
+    
+    if args["--height"]: 
+        try: height = parseInt($args["--height"]) 
+        except: echo "Warning: height must be an integer. Default value is used."
+
+    
+    #----------------------------------------------#
+    #              Defining scenary                #
+    #----------------------------------------------#
+
+    let 
+        timeStart = cpuTime()
+        a_ratio = float32(width)/float32(height)
```

Comment:
I think that the float conversion is not necessary as long as you declare the type of a_ratio.

Reply:
That's true, fixing it right now